src/modules/theme/components/chips/document-location/chips-document-location.component.html

```diff
@@ -0,0 +1,7 @@
+<ul class="nhsuk-u-padding-0">
+    <li *ngFor="let item of responseLocations" class="d-inline-block nhsuk-u-margin-1">
+      <a href="javascript:void(0)" (click)="onClickChip(item)" class="app-tag nhsuk-u-font-size-14" [ngClass]= "selectedLocations.includes(item) ? 'app-tag__selected' : ''">
```

Comment:
You can do the opposite way, I think its cleaner:

`[ngClass]="{'app-tag__selected': selectedLocations.includes(item)}"`

Reply:
Nice! done